scripts/mkcloud

```diff
@@ -110,7 +110,7 @@ iscloudver 7plus && : ${controller_node_memory:=12582912}
 : ${ironicnode_hdd_size:=20}
 
 # mysql (MariaDB actually) is the default option for Cloud8
-iscloudver 8plus && export want_database_sql_engine="mysql"
+iscloudver 8plus && : ${want_database_sql_engine:="mysql"}
```

Comment:
The line should be moved into
scripts/lib/mkcloud-common.sh
so that it will also be used for bare-metal deployments, that use only qa_crowbarsetup, but not mkcloud.
Also avoids trouble with passing it into the admin VM (you would have to `export want_database_sql_engine` )

Reply:
true, but not a problem introduced in this patch set but in the previous one..

Reply:
this patch is introducing a problem by dropping the `export`